### ISSUE TITLE
[1.3-modules-beta.4] can't use tsumina:meteor-systemjs

### ISSUE THREAD

**Issue by stphdenis** [meteor/meteor#6046]
With a fresh meteor app :
```
meteor --release METEOR@1.3-modules-beta.4 create tstV3
cd tstV3
meteor remove blaze-html-templates spacebars
meteor add tsumina:meteor-aurelia
```
on the browser platform, we get this error :
```
Uncaught TypeError: Cannot read property 'match' of undefined
```
on the line 9 of the file `system.src.js`
```
   var isWindows = typeof process != 'undefined' && !!process.platform.match(/^win/);
```
just saying that `process.platform` does not exist.
It mean's that `process` did not exist in meteor 1.2
but meteor 1.3 has now `process.env.NODE_ENV="development"`.

**Comment by benjamn** [comment]
Closing in favor of this umbrella issue: https://github.com/meteor/meteor/issues/6055

**Comment by benjamn** [comment]
Fixed by efe53de492da6df785f1cbef2799d1d2b492a939.
